### DISCUSSION TITLE
Is it possible to only publish specific parts of the listing

### DISCUSSION THREAD

**Discussion by Themaapps** [Triple-T/gradle-play-publisher#1080] [Q&A]
Hi

Would it be possible to only publish specific listing directory?
For example, I might want to only update the phone-screenshots (for all languages) 

For now, from my understanding of the documentation, the publishListing updates all the datas, which triggered a massive validation from Google. I did not try because such an update would be a problem, I do not need to update all the aspects of the listing just for a few images.

If it is something already possible, I would be more than happy :)
Thank you for your great work
Thomas

**Comment by SUPERCILEX** [comment]
No, this isn't possible right now, but it seems like a reasonable feature request. As a hack, you could delete everything except what you want to publish and then run the task.

**Reply by Themaapps** [reply]
It would be great if this can be implemented: to have separate publishlistings for each graphics (phone-screenshots, wear-screenshots, icon, feature-graphic), and for descriptions.

Your hack is a good trick, but would it not cancel the whole task because somes items are missing? Only the available items are updated ?
If only the full-description.txt remains, only this part will be published?
For graphics, does the task delete all graphics before publishing new ones? or is it replacing by name (3.png <>3.png)?

**Reply by SUPERCILEX** [reply]
My memory is fuzzy, but I believe it will only publish one type at a time. So you can't just have a full description, you'd also need whatever the other txt files are, but it shouldn't touch other stuff. Similarly, it'll publish all phone screenshots, but not touch the other folders for example. Though honestly I don't remember the details so I'd recommend playing around with a dummy app or modifying the plugin.
